lib/task.js

```diff
@@ -273,9 +275,43 @@ export class Task {
         .concat(oldEntrypoint);
     }
 
+    if(this.task.payload.features && this.task.payload.features.allowPtrace) {
```

Comment:
I think this unfortunately is the easiest least intrusive way of adding this.  In the future as we add more features, I want to create some transformation process for the docker config so it passes the config to each feature handler and it will return a modified config based on whatever it needs to do.